tensorflow/lite/micro/examples/person_detection/BUILD

```diff
@@ -16,13 +18,49 @@ cc_library(
     ],
 )
 
+generate_cc_arrays(
+    name="generated_no_person_bmp_cc",
+    src="testdata/no_person.bmp",
+    out="testdata/no_person_image_data.cc",
+)
+
+generate_cc_arrays(
+    name="generated_no_person_bmp_hdr",
+    src="testdata/no_person.bmp",
+    out="testdata/no_person_image_data.h",
+)
+
+generate_cc_arrays(
+    name="generated_person_bmp_cc",
+    src="testdata/person.bmp",
+    out="testdata/person_image_data.cc",
+)
+
+generate_cc_arrays(
+    name="generated_person_bmp_hdr",
+    src="testdata/person.bmp",
+    out="testdata/person_image_data.h",
+)
+
+generate_cc_arrays(
+    name="generated_person_detect_model_cc",
+    src="person_detect.tflite",
+    out="person_detect_model_data.cc",
+)
+
+generate_cc_arrays(
+    name="generated_person_detect_model_hdr",
+    src="person_detect.tflite",
+    out="person_detect_model_data.h",
+)
```

Comment:
since we use these in both benchmarks and examples, would this PR be a good point in time to centralize such artificats? maybe a top-level data directory?

Reply:
Done.